src/core/Tooltip/style.ts

```diff
@@ -91,13 +92,14 @@ export const tooltipCss = (props: ExtraProps): string => {
 };
 
 export const arrowCss = (props: ExtraProps): string => {
-  const { inverted, sdsStyle } = props;
+  const { inverted, sdsStyle, arrowOffset } = props;
 
   const borders = getBorders(props);
 
   return css`
     &.MuiTooltip-arrow {
       color: ${inverted || sdsStyle === "dark" ? "black" : "white"};
+      left: ${arrowOffset}px !important;
```

Comment:
can we create a more specific selector rather than using `!important`?

Reply:
@mmmmmmaya correct me if I'm wrong but I think I need it here to overwrite the `left` styles applied at rendering, happy to look into it more if not 
![Screen Shot 2022-05-19 at 2 20 55 PM](https://user-images.githubusercontent.com/36094225/169396850-f50adee5-4e43-4c67-9e32-ab7423f2f8ea.png)